lib/cli/ui/frame/frame_style/box.rb

```diff
@@ -140,10 +140,12 @@ def edge(text, color:, first:, right_text: nil)
               o << "\r"
 
               o << color.code
```

Comment:
Can you check this is correct and then add it or similar? I found this change took a lot to parse.
```suggestion
              #    preamble_start                    suffix_start
              #    |            preamble_end         |          suffix_end
              #    |            |                    |          | termwidth
              #    |            |                    |          | |
              #    V            V                    V          V V
              # ---Preamble text---------------------suffix text---
              o << color.code
```